webservice/src/main/resources/app/scripts/containers/Streams/TopologyEditor/TopologyEditorContainer.jsx

```diff
@@ -1480,8 +1495,15 @@ class TopologyEditorContainer extends Component {
         </Modal>
 
         {/*ConfirmBox to Run TestCase*/}
-        <Modal ref="confirmRunTestModal" data-title="Confirm Box" dialogClassName="confirm-box" data-resolve={this.confirmRunTest.bind(this, true)} data-reject={this.confirmRunTest.bind(this, false)}>
-          {<p> Are you sure you want to run the test case?</p>}
+        <Modal ref="confirmRunTestModal" data-title="Are you sure you want to run the test case?" data-resolve={this.confirmRunTest.bind(this, true)} data-reject={this.confirmRunTest.bind(this, false)}>
```

Comment:
nit: Change the title message to "Are you sure you want to run the test case with the following configuration?"